<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddInvoice_Dialog</class>
 <widget class="QDialog" name="AddInvoice_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_Exit">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_Save">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea_Workspace">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>497</width>
        <height>896</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="8" column="3">
        <widget class="QPushButton" name="pushButton_SetStorageRecipient">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Отправитель:</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0" colspan="3">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Сумма оплаты:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_StorageSender">
         <property name="text">
          <string>не задано</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="6">
        <widget class="QTableWidget" name="tableWidget_Products">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <column>
          <property name="text">
           <string>Номенклатура</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Наименование</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ед. изм.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Количество</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Цена покупки</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Цена продажи</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Сумма покупки</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Сумма продажи</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="16" column="0" colspan="6">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Оплата</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_StorageRecipient">
         <property name="text">
          <string>не задано</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="pushButton_SetSender">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="6">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="6">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Отправитель/Получатель/Склады</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_NameRecipient">
         <property name="text">
          <string>не задано</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Склад получатель:</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Тип оплаты:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QPushButton" name="pushButton_ClearRecipient">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item row="12" column="4">
        <widget class="QPushButton" name="pushButton_DeleteProduct">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="6">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_NameSender">
         <property name="text">
          <string>не задано</string>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QPushButton" name="pushButton_ClearStorageRecipient">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="6">
        <widget class="QComboBox" name="comboBox_TypeInvoice">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Приход</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Уход</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Перемещение</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="15" column="0" colspan="6">
        <widget class="QPlainTextEdit" name="plainTextEdit_Note">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QPushButton" name="pushButton_ClearStorageSender">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="6">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="6">
        <widget class="QLabel" name="label_NumberInvoice">
         <property name="font">
          <font>
           <pointsize>21</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Накладная №</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="6">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="5">
        <widget class="QPushButton" name="pushButton_AddProduct">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_TypePayment">
         <item>
          <property name="text">
           <string>Наличный</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Безналичный</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="0" colspan="6">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Товары</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="6">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="pushButton_SetRecipient">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="6">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Заметки</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="pushButton_ClearSender">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QPushButton" name="pushButton_SetStorageSender">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Получатель:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="3">
        <widget class="QPushButton" name="pushButton_SetProduct">
         <property name="text">
          <string>Изменить</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Склад отправитель:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="19" column="3" colspan="3">
        <widget class="QLabel" name="label_SumPayment">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="20" column="0" colspan="3">
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Дата:</string>
         </property>
        </widget>
       </item>
       <item row="20" column="3" colspan="3">
        <widget class="QDateEdit" name="dateEdit_Date">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
